### ISSUE TITLE
Cannot find native module (node-webrtc)

### ISSUE THREAD

**Issue by Zentendo** [vercel/pkg#364]
Has anyone gotten this to work with node-webrtc? I put the ```wrtc.node, webrtc.lib, webrtc.exp``` files into the same file as the compiled .exe (Windows) and it still says ```Unable to find native module``` when I run it.

**Comment by jorgexgb** [comment]
Hi, Im having this same issue. Anyone figure this one out?

**Comment by jorgexgb** [comment]
@Zentendo I found the issue to this. The issue lies in how node-cmake finds the built native add-ons. 

If you go `index.js` inside of `node_modules/node-cmake` you will see the issue.
I was able to get a temporary solution by overwriting the `binding.js` file in `wrtc` module.

This file is located in `node_modules/wrtc/lib/binding.js` and I replace its contents just before calling `pkg` with the following:

```
'use strict';
const path = require('path');

module.exports = require(path.join('..', 'build', 'Release', 'wrtc.node'));
```

This worked for me!
